src/eventcontroller/DefaultEventController.ts

```diff
@@ -96,7 +135,38 @@ export default class DefaultEventController implements EventController {
       await this.eventReporter?.reportEvent(timestampMs, name, flattenedAttributes);
     } catch (error) {
       /* istanbul ignore next */
-      this.audioVideoController.logger.error(`Error reporting event ${error}`);
+      this.logger.error(`Error reporting event ${error}`);
+    }
+  }
+
+  /** @internal */
+  private setupEventReporter(
+    configuration: MeetingSessionConfiguration,
```

Comment:
Since this is to decouple event controller from meeting in general, should we pass in a meeting session configuration object at all? Can we event client configuration? We can have a factory method in event client configuration to create it from a meeting session configuration.
@devalevenkatesh what do you think

Reply:
We could for the setupEventReporter, but I don't think we'd be able to get the meta data we sent as part of our events from event client configuration